aws/resource_aws_glue_crawler_test.go

```diff
@@ -1575,102 +1577,104 @@ resource "aws_glue_catalog_database" "test" {
 }
 
 resource "aws_s3_bucket" "default" {
-  bucket = %[1]q
+  bucket        = %[1]q
   force_destroy = true
 }
 
 resource "aws_glue_catalog_table" "test" {
-	count = %[2]d
+  count = %[2]d
 
-	database_name = "${aws_glue_catalog_database.test.name}"
-	name          = "%[1]s_table_${count.index}"
-	table_type    = "EXTERNAL_TABLE"
+  database_name = aws_glue_catalog_database.test.name
+  name          = "%[1]s_table_${count.index}"
+  table_type    = "EXTERNAL_TABLE"
 
-	storage_descriptor {
-		location      = "s3://${aws_s3_bucket.default.bucket}"
-	}
+  storage_descriptor {
+    location = "s3://${aws_s3_bucket.default.bucket}"
+  }
 }
 
 resource "aws_glue_crawler" "test" {
   depends_on = ["aws_iam_role_policy_attachment.test-AWSGlueServiceRole"]
 
-  database_name = "${aws_glue_catalog_database.test.name}"
+  database_name = aws_glue_catalog_database.test.name
   name          = %[1]q
-  role          = "${aws_iam_role.test.name}"
+  role          = aws_iam_role.test.name
 
   schema_change_policy {
     delete_behavior = "LOG"
   }
 
   catalog_target {
-    database_name = "${aws_glue_catalog_database.test.name}"
-    tables = flatten(["${aws_glue_catalog_table.test[*].name}"])
+    database_name = aws_glue_catalog_database.test.name
+    tables        = flatten([aws_glue_catalog_table.test[*].name])
   }
 
   configuration = <<EOF
 {
-  "Version":1.0,
+  "Version": 1,
```

Comment:
Interesting, I would've expected the API to complain about this change or Terraform to report it as a difference, but neither did:

```
--- PASS: TestAccAWSGlueCrawler_Configuration (108.21s)
```